### ISSUE TITLE
Restating the Legislative Power Article

### ISSUE THREAD

**Issue by KacperZajc** [liberland/constitution#183]
Time to rewrite the Legislative Power Article. 
XXX stand for a relevant provision until we are sure we are not adding any new sections here :)

Article III: The Legislative Power
•	§III.1. The legislative power in the Free Republic of Liberland shall be vested in the Citizens of the Free Republic of Liberland and the Assembly of the Free Republic of Liberland which shall exercise it on behalf of the Citizens.
•	§III.1(1) The Assembly shall consist of twenty Assembly Representatives elected by the Citizens in the General Election held every four years; 
•	§III.1(2) No person shall hold the office of a judge of one of the courts of the Free Republic of Liberland and be an Assembly Representative at the same time.
•	§III.1(3) No Assembly Representative shall be precluded from taking part in any Assembly vote by virtue of being detained prior to one's trial;
•	§III.1(4) No Assembly Representative shall hold the office for more than eight years in total.
•	§III.2. One-fourth of all Assembly Representatives shall have the right to propose a resolution dissolving the Assembly.
•	§III.2(1) The resolution shall be passed with a majority of two-thirds of all Assembly Representatives.
•	§III.2(2) Should the Assembly be dissolved, a general election shall be held within six weeks.
•	§III.3. The Assembly shall regulate its own affairs with a simple majority vote of all Assembly Representatives.
•	§III.3(1) The Assembly shall elect the Assembly Speaker, who shall preside over the business of the Assembly.
•	§III.4. One-fourth of all Assembly Representatives shall have the right to propose a resolution instituting the Assembly Committee.
•	§III.4(1) The resolution shall be passed with a majority of two-thirds of all Assembly Representatives.
•	§III.4(2) The Assembly Committee shall be composed of five Assembly Representatives.
•	§III.4(3) The Assembly Committee shall be established upon a suspicion of misconduct or mismanagement of any Member or Agent of the Public Administration.
•	§III.4(4) The Assembly Committee shall have the right to summon Members and Agents of the Public Administration to hear testimonies given under oath.
•	§III.4(5) The Assembly Committee shall prepare the Final Conclusions reporting evidence (if any) of said misconduct or mismanagement; such conclusions shall not have any legal effect. 
•	§III.4(6) The Final Conclusions shall be made public as well as passed to the Chief Public Prosecutor for consideration of potential criminal prosecution of relevant persons as described by the Committee; the Conclusions shall not bind the Chief Public Prosecutor to institute any prosecution.
•	§III.4(7) No person shall be penalised by the Assembly Committee; testimonies given in breach of the oath shall be prosecuted in regular manner.
•	§III.5. The powers of the Assembly shall be restricted to the following only:
•	§III.5(1) to protect the rights of the people to their persons and property;
•	§III.5(2) to conduct the financial affairs of the Public Administration;
•	§III.5(3) to lay and collect taxes provided for in the Taxation Bill for one or more of the following purposes only:
•	§III.5(3)(a) covering the expenses of the Public Administration;
•	§III.5(3)(b) maintaining the internal security;
•	§III.5(3)(c) providing for the system of Administration of Justice;
•	§III.5(3)(d) securing and promoting the interests of the Free Republic of Liberland worldwide.
•	§III.5(4) to establish a militia to maintain the internal security;
•	§III.5(5) to maintain the system of the Administration of Justice;
•	§III.5(6) to make such laws as shall be necessary for carrying into execution the powers vested by this Constitution in the Assembly provided that no law imposes greater restrictions on the people than absolutely necessary for the attainment of the end;
•	§III.5(7) to assent to the incorporation of any new territories to the Free Republic of Liberland with any status the Assembly shall determine;
•	§III.5(8) to debate on matters important to the well-being of the state;
•	§III.5(9) to ratify international treaties signed by the Cabinet;
•	§III.5(10) to establish the Assembly Committee;
•	§III.5(11) to assent to classification of information for the purposes of national security as proposed by a Secretary of State.
•	§III.6. Any Bill proposed to the Assembly shall pertain to one matter only as expressed in its title.
•	§III.6(1) The Bill shall contain rationale behind it and means of achieving thereof including precise calculations as to its potential impact on the budget.
•	§III.6(2) The Bill shall specify under which power delegated by this Constitution to the Assembly it is proposed.
•	§III.6(3) The Bill shall be read aloud in its entirety by the Assembly Speaker before it is voted on.
•	§III.7. Any Bill proposed to the Assembly by the Cabinet upon the State of the Republic Speech, shall be debated on, and shall be voted on once only.
•	§III.7(1) The Bill proposed by the Cabinet shall be made public prior to the Assembly's debate on them.
•	§III.7(2) All drafts of Bills proposed to the Cabinet shall be made public before the Cabinet’s acceptance or rejection.
•	§III.8. Every Assembly Representative shall have the right to express one's opinion on the proposed Bill at least once before the vote takes place, without any time limit.
•	§III.9. The Assembly shall elect the Chancellor of the Free Republic of Liberland, who shall form the Cabinet of the Free Republic of Liberland with a simple majority vote of all Assembly Representatives.
•	§III.9(1) All members of the Cabinet shall be Citizens of the Free Republic of Liberland.
•	§III.9(2) If no Chancellor is elected for more than ninety days, the General Election must be held.
•	§III.9(3) The Assembly shall have the power to withdraw confidence from the Chancellor along with the Cabinet by passing the constructive motion of no confidence with a simple majority vote of all Assembly Representatives only if there is a majority for a prospective successor
•	§III.9(4) The Assembly shall have the power to withdraw confidence from a single Member of the Cabinet by passing the ordinary motion of no confidence with a simple majority vote of all Assembly Representatives.
•	§III.10. The power to initiate a legislative procedure shall lie with the Cabinet only which shall be exercised only once a year upon the State of the Republic speech.
•	§III.10(1) Any Bill which is not voted on before the next State of the Republic Speech and any Bill which is rejected by the Assembly in a vote shall be considered lost and shall not be revived otherwise than by resubmission by the Cabinet upon another State of the Republic Speech.
•	§III.11. One-fourth of all Assembly Representatives shall have the right to oblige the Cabinet to consider a draft of a Bill prepared by them.
•	§III.11(1) Drafts prepared by the Assembly Representatives might be rejected or accepted by the Cabinet as potential Bill.
•	§III.11(2) Should the Draft be rejected, the Cabinet shall publicly give detailed reasons behind its decision.
•	§III.12. An ordinary Bill shall be passed into law with a majority of three-quarters of all Assembly Representatives subject to the power of general veto by the Citizens as provided in §XXX;
•	§III.12(1) An ordinary Bill is any Bill which is not a Constitutional Bill, or Taxation Bill, or Treaty Bill, or Financial Bill.
•	§III.13. A Constitutional Bill proposing an Amendment to this Constitution shall be passed only with the unanimous vote with all Assembly Representatives present, and shall be subject to the power of general veto by the Citizens as provided in XXX.
•	§III.14. A Taxation Bill proposing taxation shall be passed into law with the unanimous vote, and shall be subject to the power of general veto by the Citizens as provided in §XXX.
•	§III.14(1) The Bill shall introduce either a tax on actual land or sales tax.
•	§III.14(2) No other form of tax shall be levied in the Free Republic of Liberland.
•	§III.14(3) No other Bill shall be capable of levying any tax in the Free Republic of Liberland.
•	§III.15. A Treaty Bill ratifying an international treaty signed by the Cabinet shall be passed into law with a majority of three-quarters of all Assembly Representatives, and shall be subject to the power of general veto by the Citizens as provided in §XXX.
•	§III.15(1) All international treaties must be signed by the Chancellor on behalf of the Cabinet and ratified by the Assembly in order to constitute the law of the land in the Free Republic in Liberland.
•	§III.16. The Financial Bill containing the provisions of the Annual Budget shall be passed with a simple majority vote of all Assembly Representatives and shall be subject to the power of general veto by the Citizens as provided in §XXX.
•	§III.16(1) The Annual Budget shall not be with deficit.
•	§III.16(2) The Annual Budget shall constitute the only source of financial provisions for the Free Republic of Liberland.
•	§III.16(3) No provision of the Annual Budget shall provide for:
•	§III.16(3)(a) any investment that would be designed mainly to boost the economy and/or create employment and/or maintain employment; nor for acquiring any property by the Public Administration for purposes other than recognised by this Constitution;
•	§III.16(3)(b) any form of direct and/or indirect bail out of any legal and/or natural person;
•	§III.16(3)(c) funding or partial funding of any political party;
•	§III.16(3)(d) funding or partial funding of any form of healthcare and/or education services, and/or pension schemes.
•	§III.17. Any ordinary Bill or Treaty Bill or Financial Bill passed by the Assembly under this Constitution shall be subject to the general right of veto by the Citizens as expressed in the referendum.
•	§III.17(1) The referendum shall be triggered by the request of 3% of the overall number of Citizens, and shall be submitted within ninety days after the Bill was passed by the Assembly.
•	§III.17(2) The Bill shall be deemed to veto the legislative proposal if the majority of the Citizens taking part in that referendum vote 'Nay'.
•	§III.17(3) Should the Financial Bill be vetoed, the provisions of the Annual Budget adopted the previous year shall apply.
•	§III.18. Any Taxation Bill or Constitutional Bill shall be subject to the mandatory referendum to take place within ninety days after the Bill was passed by the Assembly, 
•	§III.18(1) The Bill shall be deemed to be vetoed unless a majority of two-thirds of all those taking part in a referendum vote 'Aye'.
•	§III.19 Any Bill vetoed by the Citizens in the referendum shall be considered void and shall not be revived for at least 3 years.
•	§III.20. A Bill shall become ‘under consideration’ once it is passed by the Assembly in accordance with the provisions of this Article, and either 90 days have passed and no referendum was called in accordance with §XX, or on the day the Bill obtained the assent of the Citizens in referendum under §XXX or §XXX.
•	§III.21. A Bill shall become the law of the Free Republic of Liberland, and shall be known as “Act” once it became ‘under consideration’ in accordance with the procedure provided for in §XXX, and was declared constitutional by the Supreme Court of the Free Republic of Liberland.
•	§III.22. The law of the Free Republic of Liberland shall enter into force no sooner than on the 31st day after it is signed by the Chief Justice of the Supreme Court, or if unable, by any other Justice of the Supreme Court, which shall take place upon the declaration of constitutionality.

any comments?

**Comment by terrorist96** [comment]
> • §III.3(1) The Assembly shall elect the Assembly Speaker, who shall preside over the business of the Assembly**.**

;
> • §III.4(3) The Assembly Committee shall be established upon a suspicion of misconduct or mismanagement of any **M**ember of the Public Administration.

Do we want to include this only for Members of the PA or all Agents of the PA?

> • §III.4(4) The Assembly Committee shall have the right to summon **M**embers and **A**gents of the Public Administration to hear testimonies given under oath.

;
> • §III.5(11) to assent to classification of information for the purposes of national security as proposed by a Secretary of State**.**

;
> • §III.6(3) The Bill shall be read aloud in **its** entirety by the Assembly Speaker before it is voted on.

;
> • §III.8. Every Assembly Representative shall have the right to express one's opinion on the proposed Bill at least once before the vote takes place**,** without any time limit.

;
> • §III.9. The Assembly shall elect the Chancellor of the Free Republic of Liberland**,** who shall form the Cabinet of the Free Republic of Liberland with a simple majority vote of all Assembly Representatives.

;
> • §III.14(1) The Bill shall introduce either **a** tax on actual land or sales tax.

;
> • §III.17(1) The referendum shall be triggered by the request of no less than 3% of the overall number of Citizens...

This allows for a minimum of 3% required, but doesn't specify the max. The government can interpret this as saying no less than 3%, but we will require 20%, for example. I would reword it as:
"the request of no **more** than 3%"
(Same should be applied to the Judicial section too).
The difference is that the original says at least 3% whereas my version makes 3% (or more) the requirement.

> • §III.22. The law of the Free Republic of Liberland shall enter into force no sooner than on the 31st day after it is signed by the Chief Justice of the Supreme Court, or if unable, by any other Justice of the Supreme Court**,** which **shall** take place upon the declaration of constitutionality.

**Comment by KacperZajc** [comment]
I adopted all the changes :)
